app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -98,10 +98,15 @@ def had_barcode_attention_result?
     end
 
     def redo_document_capture_pending?
+      return true if !session_result && document_capture_session&.requested_at.present?
       return unless session_result&.dig(:captured_at)
```

Comment:
we may take a detailed look, is it necessary? what's our assumption when this method is invoked? what are the edge cases?